integration_versions.md

```diff
@@ -0,0 +1,52 @@
+# This is a table of supported integration versions generated from gemfiles.
+
+| Integration | Ruby Min |  Ruby Max | JRuby Min | JRuby Max |
+|-------------|----------|-----------|----------|----------|
+| action_cable | 5.2.8.1 | 7.1.3.4 | 5.2.8.1 | 6.1.7.8 |
```

Comment:
My suggestion for excluded versions (e.g. `!= 3.7.0`) is to add an extra column (Ruby Unsupported & JRuby Unsupported) and add these individually excluded versions in it.

Reply:
@marcotc Are there currently versions that need are excluded within supported ranges of gems or is this something that would be a nice to have?